### ISSUE TITLE
nczarr_test/run_cachetest fails

### ISSUE THREAD

**Issue by WardF** [Unidata/netcdf-c#2948]
The following is observed when running `ctest` if `NETCDF_ENABLE_LARGE_FILE_TESTS=TRUE` is passed to `cmake` at configure time.

```
226/245 Test #226: nczarr_test_run_cachetest .................***Failed   13.32 sec
findplugin.sh loaded
final HDF5_PLUGIN_DIR=/Users/wfisher/Desktop/workspaces/v4.9.3-prep/netcdf-c-v4.9.3-rc1/build/plugins
*** Test: cache operation
*** write cache
write: chunkSize=131072, numCells=6553600, numSteps=360, filename=file://tmp_cachetest.zarr#mode=zarr,file
default chunk cache: size=16777216, nelems=1000, preemption=0.75
set chunk cache: size=26214400, nelems=1000, preemption=0.5
Max mem: 118 MB
*** Failed: used: 118MB expected: < 100MB
```

I'll dig into it, doesx anything leap out at you @DennisHeimbigner ?

**Comment by DennisHeimbigner** [comment]
sorry, no idea.

**Comment by WardF** [comment]
Fixed by bumping expected cache in 6aacb1b7aeac5d914fbbe136d4992a157161a164.  I *think* this is the result of different default block sizes on different platforms.
